### ISSUE TITLE
ITV: Video production has no renditions for DotCom platform

### ISSUE THREAD

**Issue by CalebWhiting** [rg3/youtube-dl#14944]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2017.12.02*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2017.12.02**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-v', u'https://www.itv.com/hub/through-the-keyhole/2a2271a0033']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2017.12.02
[debug] Python version 2.7.14 - Linux-4.14.0-041400-generic-x86_64-with-Ubuntu-17.10-artful
[debug] exe versions: ffmpeg 3.4, ffprobe 3.4, rtmpdump 2.4
[debug] Proxy map: {}
[debug] Using fake IP 25.62.55.185 (GB) as X-Forwarded-For.
[ITV] 2a2271a0033: Downloading webpage
[ITV] 2a2271a0033: Downloading XML
ERROR: ITV said: Video production has no renditions for DotCom platform.; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 784, in extract_info
    ie_result = ie.extract(url)
  File "/bin/youtube-dl/youtube_dl/extractor/common.py", line 437, in extract
    ie_result = self._real_extract(url)
  File "/bin/youtube-dl/youtube_dl/extractor/itv.py", line 114, in _real_extract
    raise ExtractorError('%s said: %s' % (self.IE_NAME, fault_string))
ExtractorError: ITV said: Video production has no renditions for DotCom platform.; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
...
<end of log>
```

### Description of your *issue*, suggested solution and other information

For some reason this is only happening for me with this particular video, I've manually verified that the video will playback in the browser.

**Comment by CalebWhiting** [comment]
Just to clarify, I'm a UK resident, my IP address is not geo-restricted, from the log I assume it's trying to connect with a proxy, however there should be no need.

**Comment by ravonel** [comment]
I've started to hit this problem more often as well, and I don't think it has anything to do with geo-restriction. It seems to do with the playlist service that youtube-dl's current ITV extractor tries to access, at https://secure-mercury.itv.com/PlaylistService.svc?wsdl
When I try to display the returned XML for the videos that won't download, I get:
`<s:Envelope xmlns:s="http://schemas.xmlsoap.org/soap/envelope/"><s:Body><GetPlaylistResponse xmlns="http://tempuri.org/"><GetPlaylistResult><Fault xmlns=""><faultcode>InvalidEntity</faultcode><faultstring>Video production has no renditions for DotCom platform.</faultstring></Fault></GetPlaylistResult></GetPlaylistResponse></s:Body></s:Envelope>`
as opposed to the regular playlist with all the video and ad and whatever information.

That hasn't been for all the ITV videos, it just seems to be a tiny but annoying fraction, heh.
However, those videos all still play in the browser, so something must be working right. Looking at the network requests, it seems maybe there's a shift to a new playlist service? For your vid, that would be: https://magni.itv.com/playlist/itvonline/ITV/2_2271_0033.001
The relevant info seems to be there:
```
{
  "StatusCode" : 200,
  "AdditionalInfo" : {
    "TransactionId" : "oas-magni-360304-p5GnWf",
    "Message" : null
  },
  "Playlist" : {
    "VideoType" : "CATCHUP",
    "ContentBreaks" : [
      {
        "TimeCode" : "00:00:00:000",
        "Actions" : [
          {
            "Type" : "ident",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videoident/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=0/adpos=iden1/viewid=0.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=0/adpos=1/viewid=0.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=0/adpos=last/viewid=0.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=0/adpos=sponlast1/viewid=sponlast1.0.53735df3-a490-41a1-8213-0c48f179c9fe"
          }
        ]
      },
      {
        "TimeCode" : "00:17:40:240",
        "Actions" : [
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=spon1/viewid=spon1.1.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=1/viewid=1.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=2/viewid=1.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=3/viewid=1.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=4/viewid=1.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=last/viewid=1.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=1/adpos=sponlast1/viewid=sponlast1.1.53735df3-a490-41a1-8213-0c48f179c9fe"
          }
        ]
      },
      {
        "TimeCode" : "00:27:04:920",
        "Actions" : [
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=spon1/viewid=spon1.2.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=1/viewid=2.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=2/viewid=2.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=3/viewid=2.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=4/viewid=2.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=last/viewid=2.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=2/adpos=sponlast1/viewid=sponlast1.2.53735df3-a490-41a1-8213-0c48f179c9fe"
          }
        ]
      },
      {
        "TimeCode" : "00:35:59:680",
        "Actions" : [
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=spon1/viewid=spon1.3.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=1/viewid=3.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=2/viewid=3.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=3/viewid=3.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=4/viewid=3.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=last/viewid=3.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=3/adpos=sponlast1/viewid=sponlast1.3.53735df3-a490-41a1-8213-0c48f179c9fe"
          }
        ]
      },
      {
        "TimeCode" : "00:47:13:960",
        "Actions" : [
          {
            "Type" : "sponsor",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=videosponsor/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=end/adpos=spon1/viewid=spon1.end.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=end/adpos=1/viewid=end.53735df3-a490-41a1-8213-0c48f179c9fe"
          },
          {
            "Type" : "advert",
            "Href" : "https://tom.itv.com/itv/tserver/pm=free/random=1981026377430494553/area=itvplayer.video/source=catchup/hdevid=x/size=video/supertag=dflt,itv.2_2271_0033/us=ano/site=itv/osver=6.1/chanbrand=itv1/os=windows.nt/hmod=62/pv=browser.4.1/player=html5.desktop/dm=8efe5785-6229-42e4-9918-7bd41ea5a282/hman=chrome/tdur=2833/tparts=4/arp=none/plist=i1.s1.pr2.m5.po2/plfcid=57597/linked_fcid=57597/position=x13707769290688629709/platform=desktop/generic=53735df3-a490-41a1-8213-0c48f179c9fe/breaknum=end/adpos=last/viewid=end.53735df3-a490-41a1-8213-0c48f179c9fe"
          }
        ]
      }
    ],
    "ProductionId" : "2/2271/0033#001",
    "Dog" : {
      "Href" : "https://magni.itv.com/img/itv.png"
    },
    "Video" : {
      "Duration" : "00:47:13:960",
      "Subtitles" : [
        {
          "Href" : "https://itvpnpsubs-a.akamaihd.net/2-2271-0033-001/Subtitles/1/WebVTT-OUT-OF-BAND/2-2271-0033-001_ThroughtheKeyholeSeries5_TX091217.vtt"
        }
      ],
      "Base" : "https://itvpnpdotcom-a.akamaihd.net/",
      "MediaFiles" : [
        {
          "Href" : "2-2271-0033-001/18/1/VAR011/2-2271-0033-001_18_1_VAR011.ism/.mpd?hdnea=st=1513091350~exp=1513112950~acl=/*~hmac=13c038ae3549a09a864295735957c6627be0e40a1856b31adfd09803f268c512",
          "KeyServiceUrl" : "https://itvpnp.live.ott.irdeto.com/ClearKey/getkeys?CrmId=itvpnp&AccountId=itvpnp&ContentId=2-2271-0033-001_18"
        }
      ],
      "Token" : null
    }
  }
}
```

**Comment by ravonel** [comment]
To clear up some information I might have been wrong on, I guess secure-mercury continues to be used, if you download the current pages, it's still the playlist link, but I guess there's some backend entry that maybe somebody has to manually add each video to? I've had videos fail to download with the same above error that then work a few days later when a newer episode has been added, I assume someone realizes they left something off. And that's not always the case, I tried a couple of my videos that haven't worked for weeks and Caleb's link above, still not working.
